cmd/bucket-handlers.go

```diff
@@ -1685,7 +1686,12 @@ func (api objectAPIHandlers) ResetBucketReplicationStateHandler(w http.ResponseW
 
 	vars := mux.Vars(r)
 	bucket := vars["bucket"]
-	durationStr := r.Form.Get("older-than")
+	durationStr := r.URL.Query().Get("older-than")
+	arn := r.URL.Query().Get("arn")
+	resetID := r.URL.Query().Get("reset-id")
+	if resetID == "" {
+		resetID = mustGetUUID()
+	}
```

Comment:
so it has to be a UUID what is the data format for resetID? - there is no validation here for what user can send.. can you add some validation for it?

Reply:
User does not generate uuid - this is generated by minio-go API.

Reply:
should you add validation here for reset-id? what is the format for reset-id?

Reply:
a non empty string is all that is needed for reset-id - the reason I am not adding any other format check is because the reset id is generated as a uuid inside minio-go ResetBucketReplication API call, not by the end user.

Reply:
okay @poornas